### DISCUSSION TITLE
Cant find GPT4o in the model list.

### DISCUSSION THREAD

**Discussion by FernandoLLB** [danny-avila/LibreChat#2748] [Troubleshooting] [answered]
Am I missing something here? I have access to the rest of the models as usual but cant find GPT4o anywhere. The API Key is working perfectly but I dont see it. Could you guys give me a hand? 

Thank you in advance

**Comment by danny-avila** [comment]
Are you using the `OPENAI_MODELS` environment variable? If not, and it doesn't seem to be fetching, you can also add it to your list

**Comment by FernandoLLB** [comment]
I just installed docker and git and followed the installation instructions. I have not modified anything and honestly I don’t know how to do it.

Enviado desde Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: Danny Avila ***@***.***>
Enviado: Thursday, May 16, 2024 2:32:23 PM
Para: danny-avila/LibreChat ***@***.***>
Cc: Ferlok ***@***.***>; Author ***@***.***>
Asunto: Re: [danny-avila/LibreChat] Cant find GPT4o in the model list. (Discussion #2748)


Are you using the OPENAI_MODELS environment variable? If not, and it doesn't seem to be fetching, you can also add it to your list

**Reply by danny-avila** [reply]
Have you edited the `.env` file in the project folder at all? that would be a good place to start looking.

**Comment by FernandoLLB** [comment]
Than you, I found the file and oppened it but when I scroll down to the open ai section, everything looks like it should be, GPT4o appears there also. 
![Screenshot 2024-05-16 155824](https://github.com/danny-avila/LibreChat/assets/103178820/da3f4b71-5a2c-46ee-adfd-fa10e7f4e903)

Do you see something wrong?

**Comment by FernandoLLB** [comment, marked as answer]
For anyone looking for a solution to this: You have to go to the .env file and delete the "#" symbol from "#OPENAI_MODELS=" then save the file and it should work.

**Reply by rostislaved** [reply]
Why is it commented by default?

**Reply by fuegovic** [reply]
> Why is it commented by default?

If the API key is provided in the .env, and the model list is commented out, the available models will be automatically fetched from OpenAI
